dev-aws/kafka-shared/iam/iam.tf

```diff
@@ -233,3 +233,28 @@ resource "kafka_topic" "iam_revoked_v1" {
     "cleanup.policy"    = "delete"
   }
 }
+
+
+resource "kafka_topic" "iam_credentials_public_events_mapper_v1" {
+    name = "auth-customer.credentials-public-events-mapper-v1"
+    replication_factor = 3
+    parititions = 10
+    config = {
+        # retain 100MB on each partition
+        "retention.bytes" = "104857600"
+        # keep data for 7 days
+        "retention.ms" = "604800000"
+        # allow max 1 MB for a message
+        "max.message.bytes" = "1048576"
+        "compression.type"  = "zstd"
+        "cleanup.policy"    = "delete"
+    }
+}
+
+module "iam_password_change_processor" {
```

Comment:
Maybe rename the module to the app name? e.g. `iam_credentials_public_event_mapper...`